inlong-tubemq/tubemq-server/src/main/java/org/apache/inlong/tubemq/server/master/nodemanage/nodeconsumer/ConsumerEventManager.java

```diff
@@ -144,9 +144,10 @@ public ConsumerEvent removeFirst(String consumerId) {
             }
         }
         if (event != null) {
-            StringBuilder sBuilder = new StringBuilder(512);
-            sBuilder.append("[Event Removed] ");
-            logger.info(event.toStrBuilder(sBuilder).toString());
+            logger.info(new StringBuilder(512)
```

Comment:
Suggested using the `{}` instead of StringBuilder in log info.

Reply:
This is deliberately handled in this way, using StringBuilder to do the processing of log output content